### ISSUE TITLE
vim.lsp.get_active_clients() is deprecated, use vim.lsp.get_clients() instead

### ISSUE THREAD

**Issue by jiangyinzuo** [ray-x/go.nvim#459]
NVIM v0.11.0-dev will report the following error:
```
vim.lsp.get_active_clients() is deprecated, use vim.lsp.get_clients() instead. :help deprecated
Feature will be removed in Nvim 0.12
stack traceback:
        /usr/share/nvim/runtime/lua/vim/lsp.lua:785: in function 'get_active_clients'
        /root/plugged/go.nvim/lua/go/lsp.lua:182: in function 'client'
        /root/plugged/go.nvim/lua/go/inlay.lua:318: in function 'set_inlay_hints'
        /root/plugged/go.nvim/lua/go/inlay.lua:69: in function ''
        vim/_editor.lua: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

**Comment by ray-x** [comment]
I will need to refresh this later as this API is not available in 0.9.x. 
Maybe when 0.10.1 is out.

**Comment by DArcMattr** [comment]
I have a fix candidate in #461

**Comment by jiangyinzuo** [comment]
This commit [8008d16](https://github.com/ray-x/go.nvim/commit/8008d16951ab1213cb3e09c12994e758a5694fa9) works well for me. Thanks!
